### DISCUSSION TITLE
How to change the hotkey for multi-cursor?

### DISCUSSION THREAD

**Discussion by Bill-Gots** [texstudio-org/texstudio#3691] [Q&A]
I'm using Texstudio on Windows and I want to modify the hotkey for multi-cursor (Ctrl+Alt+Click), but I just can't find the setting.

**Comment by octaeder** [comment]
Main menu Options->Configure TeXstudio->Shortcuts->Editor->Basic Key Mappings->Create cursor mirror down

**Reply by Bill-Gots** [reply]
Thanks for your reply! But what I actually want to modify is the hotkey for creating a cursor at anywhere (which is Ctrl+Alt+Click).

**Comment by sunderme** [comment]
you can't change modifier key for mouse clicks.

**Reply by Bill-Gots** [reply]
😭
